.gitignore

```diff
@@ -18,4 +18,3 @@ maven-eclipse.xml
 atlassian-ide-plugin.xml
 test-output
 /bin
-report/
```

Comment:
I believe we still want to ignore the generated `report/` dir.

Reply:
yes. you're right. will fix.